kolibri/tasks/api.py

```diff
@@ -120,14 +121,22 @@ def startlocalimport(self, request):
         """
         Import a channel from a local drive, and copy content to the local machine.
         """
-        # Importing django/running setup because Windows...
 
         if "drive_id" not in request.data:
             raise serializers.ValidationError(
-                "The 'drive_id' field is required.")
+                "The 'drive_id' field is required."
+            )
+
+        if "channel_id" not in request.data:
+            raise serializers;ValidationError(
+                "The 'channel_id' field is required."
+            )
+
+        channel_id = request.data["channel_id"]
+        node_ids = request.data.get("node_ids")
```

Comment:
If you use my suggestion above, you would probably have to do the split here though!